go/vt/throttler/throttler_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Vitess Authors.
+Copyright 2020 The Vitess Authors.
```

Comment:
If this is an existing file the copyright date should not be changed.

Reply:
It's a new file. I copied it from another file, and this is why `git` felt like I was editing one.